curriculum/challenges/english/14-responsive-web-design-22/build-a-product-landing-page-project/build-a-product-landing-page.md

```diff
@@ -251,15 +251,21 @@ const cssCheck = new __helpers.CSSHelp(document).getCSSRules('media')
 assert(cssCheck.length > 0 || htmlSourceAttr.length > 0);
 ```
 
-Your Product Landing Page should use CSS Flexbox at least once.
+Your media query should contain a `display` property with the value set to `flex` or `inline-flex`
```

Comment:
I think giving such detailed instruction isn't necessary. Just let the user know they need to use flexbox.